marimo/_runtime/requests.py

```diff
@@ -33,8 +38,7 @@ class FunctionCallRequest:
     args: dict[str, Any]
 
 
-@dataclass
-class SetCellConfigRequest:
+class SetCellConfigRequest(BaseModel):
```

Comment:
I had to make these Pydantic models, Pydantic complained that `CellId_t` wasn't defined when these types were referenced in other models.